Content.Shared/Traits/Assorted/LightweightComponent.cs

```diff
@@ -0,0 +1,12 @@
+using Robust.Shared.GameStates;
+using System.ComponentModel.DataAnnotations;
+
+namespace Content.Shared.Traits.Assorted;
+
+/// <summary>
+/// Used for the lightweight trait. DrunkSystem will check for this component and modify the boozePower accordingly if it finds it.
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed class LightweightComponent : Component
```

Comment:
This should be named LightweightDrunkComponent for clarity, same as the name for the trait